server/src/test/java/org/jboss/as/server/services/security/VaultSession.java

```diff
@@ -0,0 +1,348 @@
+     *
+     * @param vaultBlock
+     * @param attributeName
+     * @param attributeValue
+     * @return secured attribute configuration
+     */
+    public String addSecuredAttribute(String vaultBlock, String attributeName, char[] attributeValue) throws Exception {
+        vault.store(vaultBlock, attributeName, attributeValue, null);
+        return securedAttributeConfigurationString(vaultBlock, attributeName);
+    }
+
+    /**
+     * Add secured attribute to specified vault block. This method can be called only after successful
+     * startVaultSession() call.
+     * After successful storage the secured attribute information will be displayed at standard output.
+     * For silent method @see addSecuredAttribute
+     *
+     * @param vaultBlock
+     * @param attributeName
+     * @param attributeValue
+     * @throws Exception
+     */
+    public void addSecuredAttributeWithDisplay(String vaultBlock, String attributeName, char[] attributeValue) throws Exception {
+        vault.store(vaultBlock, attributeName, attributeValue, null);
+        attributeCreatedDisplay(vaultBlock, attributeName);
+    }
+
+    /**
+     * Check whether secured attribute is already set for given vault block and attribute name. This method can be called only after
+     * successful startVaultSession() call.
+     *
+     * @param vaultBlock
+     * @param attributeName
+     * @return true is password already exists for given vault block and attribute name.
+     * @throws Exception
+     */
+    public boolean checkSecuredAttribute(String vaultBlock, String attributeName) throws Exception {
+        return vault.exists(vaultBlock, attributeName);
+    }
+
+    /**
+     * This method removes secured attribute stored in {@link SecurityVault}.
+     * After successful remove operation returns true. Otherwise false.
+     *
+     * @param vaultBlock security vault block
+     * @param attributeName Attribute name stored in security vault
+     * @return true is operation is successful, otherwise false
+     * @throws Exception
+     */
+    public boolean removeSecuredAttribute(String vaultBlock, String attributeName) throws Exception {
+        return vault.remove(vaultBlock, attributeName, null);
+    }
+
+    /**
+     * Display info about stored secured attribute.
+     *
+     * @param vaultBlock
+     * @param attributeName
+     */
+    private void attributeCreatedDisplay(String vaultBlock, String attributeName) {
+        System.out.println(String.format("Secured attribute value has been stored in Vault.\n" +
+            "Please make note of the following:\n" +
+            "********************************************\n" +
+            "Vault Block:%s\n" + "Attribute Name:%s\n" +
+            "Configuration should be done as follows:\n" +
+            "%s\n" +
+            "********************************************", vaultBlock, attributeName, securedAttributeConfigurationString(vaultBlock, attributeName)));
```

Comment:
You could remove the `String.format` and just use `System.out.printf`. The `\n` should probably be replaced with `%n` to be platform independent.